schemas/oic.wk.mnt-schema.json

```diff
@@ -17,6 +17,12 @@
         "rb": {
           "type": "boolean",
           "description": "Reboot Action"
+        }        ,
+        "err": {
+          "type": "integer",
```

Comment:
We need to add "readOnly": TRUE to match the CR which says that "err" is read only.

Reply:
added